lib/tasks/dev.rake

```diff
@@ -1,33 +1,30 @@
 desc "Fill the database tables with some sample data"
 task({ :sample_data => :environment }) do
+  starting = Time.now
   p "Creating sample data"
 
-  if Rails.env.development?
-    FollowRequest.destroy_all
-    Comment.destroy_all
-    Like.destroy_all
-    Photo.destroy_all
-    User.destroy_all
-  end
+ # if Rails.env.development?
```

Comment:
This line should not be commented out; Please review the "Flaw in sample_data" section in this module: https://learn.firstdraft.com/lessons/199-photogram-industrial-part-3